plugins/dicom_viewer/server/__init__.py

```diff
@@ -65,12 +65,22 @@ def coerce(x):
         return None
 
 
-def process_file(f):
+def parse_file(f):
```

Comment:
`process_file` is still there but now calls `parse_file`. I did it to have an early return.